### PR TITLE
RSLC-3 As a user I should be able to see all my targets in the map

### PR DESCRIPTION
**Pull request by diegoxanderdx**
#### Board:
* [RSLC-3](https://trello.com/c/Lnm6pxYD)
---
#### Description:
As a user I should be able to see all my targets in the map so I can easily identify them
---
#### Notes:

---
#### Tasks:
- [x] Create spec for index action
- [x] Edit policies for targets
- [x] Create index action in target controller
---
#### Risk:
*
---

### CODE REVIEW

File: app/policies/target_policy.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class TargetPolicy < ApplicationPolicy
+  def index?
```
Comment (rodrieiz) [inline app/policies/target_policy.rb:2]: I think this method unnecessary

File: spec/requests/api/v1/targets/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    let!(:user) { create(:user) }
+    let!(:user_targets) { create_list(:target, 2, user:) }
+
+    subject { get api_v1_targets_path, headers: auth_headers, as: :json }
+
+    it 'returns all targets from current user' do
+      subject
+      expect(payload).not_to be_empty
+      expect(payload.size).to eq(user_targets.size)
+      expect(response).to be_successful
```
Comment (rodrieiz) [inline spec/requests/api/v1/targets/index_spec.rb:14]: maybe split payload test and response status in another one

**Review by rodrieiz — APPROVED** [review]
👏

File: app/policies/target_policy.rb (hunk 1 above)
Comment (diegoxanderdx) [reply app/policies/target_policy.rb:2]: I'll try

File: spec/requests/api/v1/targets/index_spec.rb (hunk 2 above)
Comment (diegoxanderdx) [reply spec/requests/api/v1/targets/index_spec.rb:14]: ok. many thanks

**Review by brunoparma88 — APPROVED** [review]

**Pull request merged**